py/vm.c

```diff
@@ -1116,7 +1116,7 @@ unwind_jump:;
                     mp_uint_t unum = *ip;
                     mp_obj_t obj;
                     if (unum == 2) {
-                        mp_warning("exception chaining not supported");
+                        mp_warning(NULL, "exception chaining not supported");
```

Comment:
For full configurability NULL would need to be something like MP_WARN_NULL (eg it could be MP_OBJ_NULL if types were used instead of strings).

Reply:
> For full configurability NULL would need to be something like MP_WARN_NULL

Actually, NULLs shouldn't be there, a couple of cases where they are, I believe to not break existing tests. (The idea is to eventually fix those up).

If you think that MP_WARN_NULL would be useful, I can add it, but would be temporary (per current expectations).

Reply:
Ok, if NULL is going to be eventually fixed up then they can stay for now.